flink-tests/src/test/java/org/apache/flink/test/classloading/ClassLoaderITCase.java

```diff
@@ -51,12 +50,25 @@ public void testJobWithCustomInputFormat() {
 
 				PackagedProgram inputSplitTestProg = new PackagedProgram(new File(INPUT_SPLITS_PROG_JAR_FILE),
 						new String[] { INPUT_SPLITS_PROG_JAR_FILE,
+										"", // classpath
 										"localhost",
 										String.valueOf(port),
 										"4" // parallelism
 									} );
 				inputSplitTestProg.invokeInteractiveModeForExecution();
 
+
+
+				String classpath = new File(INPUT_SPLITS_PROG_JAR_FILE).toURI().toURL().toString();
+				PackagedProgram inputSplitTestProg2 = new PackagedProgram(new File(INPUT_SPLITS_PROG_JAR_FILE),
+						new String[] { "",
+										classpath, // classpath
+										"localhost",
+										String.valueOf(port),
+										"4" // parallelism
+									} );
+				inputSplitTestProg2.invokeInteractiveModeForExecution();
+
```

Comment:
Looks like I wrote this comment before testing the functionality ;) It seems that the test does not detect the issues I found with manual testing.